src/modules/Articles.js

```diff
@@ -0,0 +1,10 @@
+import { api } from "./network";
+
+const Articles = {
+  async index() {
+    const { data } = await api.get("/articles");
```

Comment:
I would wrap this http request in a try/catch statement, and return the error.message if something goes wrong, but it's not a real issue.